### PR TITLE
Fetch pod

### PR DESCRIPTION
**Pull request by yusufMeimeh**
Hello, 

Now you can use '**Fetch**' using Cocoapods.

All you need is to add 

`pod 'Fetch' `

in your pod file the run pod install. 

Enjoy.

### CODE REVIEW

File: Fetch.podspec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+  s.description      = <<-DESC
+  'A resource based network abstraction based on Alamofire'
+                       DESC
+  s.homepage         = 'https://github.com/allaboutapps/Fetch'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'https://www.allaboutapps.at' => 'office@allaboutapps.at' }
+  s.source           = { :git => 'https://github.com/allaboutapps/Fetch.git', :tag => s.version.to_s }
+  s.social_media_url = 'http://linkedin.com/in/yusufmeimeh'
```
Comment (mbuchetics) [inline Fetch.podspec]: please remove your linkedin page

Comment (yusufMeimeh) [reply Fetch.podspec]: done :)

**Comment by mbuchetics** [comment]
Thank you 👍

**Comment by mbuchetics** [comment]
Currently I am not the owner of the cocoapod, but I filed a claim and will update the pod as soon as I have access.

**Pull request merged**